react-app/src/components/ButtonsModal/index.js

```diff
@@ -0,0 +1,53 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+
+import BaseModal from 'components/BaseModal';
+
+import styles from './index.module.scss';
+
+export default class ButtonsModal extends PureComponent {
+  static propTypes = {
+    /**
+     * Whenever the modal is visible or not
+     */
+    visible: PropTypes.bool.isRequired,
+
+    /**
+     * The message that the modal will show
+     */
+    message: PropTypes.string.isRequired,
+
+    /**
+     * The primary button (a Button) that the element will show
+     */
+    primaryButton: PropTypes.element.isRequired,
+
+    /**
+     * The (optional) secondary button (a Button) that the element will show
+     */
+    secondaryButton: PropTypes.element,
+  };
+
+  static defaultProps = { secondaryButton: null }
+
+  render() {
+    const content = (
+      <div className={styles.container}>
+        <div>
+          {this.props.message}
+        </div>
+        <div className={styles.buttonsContainer}>
+          {this.props.secondaryButton}
+          {this.props.primaryButton}
+        </div>
+      </div>
+    );
+
+    return (
+      <BaseModal
```

Comment:
I guess in a way you're composing a BaseModal here instead of extending it... I don't know, maybe this is perfect, but my reflex was to expect ButtonsModal to extend BaseModal. So that if BaseModal has new props, you don't have to change this one.